docs/modules/ROOT/partials/nav.adoc

```diff
@@ -84,7 +84,7 @@
 ** xref:oc4:ROOT:how-tos/update_maintenance.adoc[Update/Maintenance]
 ** xref:oc4:ROOT:how-tos/update_maintenance/automated-upgrades-at-vshn.adoc[]
 ** xref:oc4:ROOT:how-tos/new_minor.adoc[Get ready for new minor]
-** xref:oc4:ROOT:how-tos/update_maintenance/v_4_9.adoc[Upgrade to OCP4.9]
+** xref:oc4:ROOT:how-tos/update_maintenance/v_4_12.adoc[Upgrade to OCP4.12]
```

Comment:
Please set up a page alias (https://docs.antora.org/antora/latest/page/page-aliases/) so that bookmarks and links on other pages do no brake.